starter-code/app.js

```diff
@@ -2,4 +2,66 @@ const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
 
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+app.set('layout', 'layouts/main-layout');
+
+
+app.use(express.static('public'));
+app.use(expressLayouts);
+
+app.get('/', (request, response, next) => {
+  response.render('index');
+});
+
+app.get('/random', (req, res, next)=>{
+  // Retrieve a random chuck joke
+  client.getRandomJoke()
+    .then((response) => {
+      res.send('<h1>' + response.value + '!</h1>');
```

Comment:
Better than send a h1, render a page with that data. You can create a view (something like random.ejs) or display it in the index.